apps/src/pcd_video_player/pcd_video_player.cpp

```diff
@@ -47,6 +47,10 @@
 #include <QMutexLocker>
 #include <QObject>
 #include <QRadioButton>
+#if VTK_MAJOR_VERSION >= 8 && VTK_MINOR_VERSION >= 2
```

Comment:
If vtk_minor_version is below 0, it can't be true. Even though the Major version is above 8.

But we can use this:

```suggestion
#if VTK_VERSION_NUMBER > VTK_VERSION_CHECK(8,2,0)
```